### PR TITLE
Add githubteacher recommendations

### LINKED ISSUE #2: githubteacher checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge

🌮

### PR DESCRIPTION
**Pull request by githubteacher**
Adding recommendations for Tampa (resolves #2 #1 ) ... for real this time.

### CODE REVIEW

File: githubteacher-rec.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Tampa, Fl
+
+### Places to Eat
+- Dunedin Brewery :taco: and :beer:
```
Comment (DavidJ-Hall) [inline githubteacher-rec.md]: @githubteacher not a :beer: person.  Any :wine_glass: suggestions?

Comment (DavidJ-Hall) [inline githubteacher-rec.md:4]: @githubteacher not a :beer: person.  Any :wine_glass: suggestions?

Comment (DavidJ-Hall) [inline githubteacher-rec.md]: @githubteacher not a :beer: person.  Any :wine_glass: suggestions?

**Review by DavidJ-Hall — COMMENTED** [review]
test?

File: githubteacher-rec.md (hunk 1 above)
Comment (githubteacher) [reply githubteacher-rec.md:4]: Hey @DavidJ-Hall made the requested change!

**Review by DavidJ-Hall — APPROVED** [review]

**Pull request merged**